tests/modules/sightings/resources/test_featured_asset_guid.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+import json
+from app.modules.sightings.models import Sighting
+from tests.modules.sightings.resources import utils as sighting_utils
+from flask import current_app
+from tests import utils
+
+
+def test_featured_asset_guid_endpoint(db, flask_app_client):
+
+    new_researcher = utils.generate_user_instance(
+        email='asset_guid_getter@mail.com', is_researcher=True
+    )
+
+    with db.session.begin():
+        db.session.add(new_researcher)
+
+    data_in = {
+        'encounters': [{}],
+        'context': 'test',
+        'locationId': 'test',
+    }
+
+    response = sighting_utils.create_sighting(
+        flask_app_client, new_researcher, data_in=data_in
+    )
+
+    sighting_id = response.json['result']['id']
+    sighting = Sighting.query.get(sighting_id)
+    assert sighting is not None
+
+    new_asset_group = utils.generate_asset_group_instance(new_researcher)
+
+    with db.session.begin():
+        db.session.add(new_asset_group)
+        db.session.add(sighting)
+
+    PATH = '/api/v1/sightings/' + str(sighting.guid) + '/featured_asset_guid'
+    SIGHTINGS_READ = 'sightings:read'
+
+    with flask_app_client.login(new_researcher, auth_scopes=(SIGHTINGS_READ,)):
```

Comment:
Doesn't the read_sighting in the sightings_utils do this??

Reply:
I thought that constructing the GET myself was a bit less hacky than concatenating '/featured_asset_guid' on to the end of the sighting guid, this is a one-off API after all. I'd venture the best way would be to allow the user to optionally override PATH in the args but since we're trying not to make custom endpoints (like I, uh, did here but will avoid going forward) very often it's probably not necessary.

Reply:
I'm actually thinking of going the other way. I think the 
def read_sighting(flask_app_client, user, sight_guid, expected_status_code=200):
template is wrong in that the sight_guid is merely a secondary part of the path, the fact that it happens to be a guid in many cases is pretty irrelevant. 
Yeah I know, I started the template so my bad :) 
If you renamed sight_guid in the read util as sub_path... or something, then you could use it and pass f'{sighting.guid}/featured_asset_guid' and could remove a fair chunk of code here and I think make it clearer exactly what's being tested.
Don't feel particularly strongly either way so as you like.